rs/tests/research/spec_compliance/spec_compliance.rs

```diff
@@ -217,16 +185,6 @@ pub fn setup_impl(env: TestEnv, deploy_bn_and_nns_canisters: bool, http_requests
             },
         );
     }
-    if deploy_bn_and_nns_canisters {
-        let cloned_env = env.clone();
-        timeit(
-            cloned_env.logger(),
-            "waiting until Boundary Node is healthy",
-            move || {
-                await_boundary_node_healthy(&cloned_env, BOUNDARY_NODE_NAME);
```

Comment:
This healthiness check has been dropped without replacement (it seems).

Reply:
I added it back in above.